cirq-core/cirq/transformers/gauge_compiling/gauge_compiling_test_utils.py

```diff
@@ -54,6 +55,27 @@ def test_gauge_transformer(self, generation_seed, transformation_seed):
                 nc, c, qubit_map={q: q for q in c.all_qubits()}
             )
 
+        # Test randomized compiling
```

Comment:
I think this block should be its own function ... for that  you can create a new test class the inherets for the parent test class and implements only the new method

Reply:
Done.

Reply:
Added a class function instead as the sweep test can be enabled for all the gauges, for unsupported gauges, expect NotImplemented error.